src/transformers/models/omdet_turbo/processing_omdet_turbo.py

```diff
@@ -0,0 +1,301 @@
+    box = torch.stack((x1, y1, x2, y2), dim=-1)
+
+    return box
+
+
+class OmDetTurboProcessorKwargs(ProcessingKwargs, total=False):
+    _defaults = {
+        "text_kwargs": {
+            "add_special_tokens": True,
+            "padding": "max_length",
+            "truncation": True,
+            "max_length": 77,
+            "stride": 0,
+            "return_overflowing_tokens": False,
+            "return_special_tokens_mask": False,
+            "return_offsets_mapping": False,
+            "return_token_type_ids": False,
+            "return_length": False,
+            "verbose": True,
+        },
+        "images_kwargs": {},
+    }
+
+
+class OmDetTurboProcessor(ProcessorMixin):
+    r"""
+    Constructs a OmDet-Turbo processor which wraps a Deformable DETR image processor and an AutoTokenizer into a
+    single processor.
+
+    [`OmDetTurboProcessor`] offers all the functionalities of [`DetrImageProcessor`] and
+    [`AutoTokenizer`]. See the docstring of [`~OmDetTurboProcessor.__call__`] and [`~OmDetTurboProcessor.decode`]
+    for more information.
+
+    Args:
+        image_processor (`DetrImageProcessor`):
+            An instance of [`DetrImageProcessor`]. The image processor is a required input.
+        tokenizer (`AutoTokenizer`):
+            An instance of ['PreTrainedTokenizer`]. The tokenizer is a required input.
+    """
+
+    attributes = ["image_processor", "tokenizer"]
+    image_processor_class = "DetrImageProcessor"
+    tokenizer_class = "AutoTokenizer"
+
+    def __init__(self, image_processor, tokenizer):
+        super().__init__(image_processor, tokenizer)
+
+    def __call__(
+        self,
+        images: ImageInput = None,
+        text: Union[str, List[str], TextInput, PreTokenizedInput] = None,
+        labels: Union[List[str], List[List[str]]] = None,
+        audio=None,
+        videos=None,
+        **kwargs: Unpack[OmDetTurboProcessorKwargs],
+    ) -> BatchEncoding:
+        """
+        This method uses [`DetrImageProcessor.__call__`] method to prepare image(s) for the model, and
+        [`CLIPTokenizerFast.__call__`] to prepare text for the model.
+
+        Please refer to the docstring of the above two methods for more information.
+        """
+        if images is None or text is None or labels is None:
+            raise ValueError("You have to specify `images`, `text` and `labels`.")
+
+        if isinstance(text, str):
+            text = [text]
+        if isinstance(labels[0], str):
+            labels = [labels]
+
+        # error when using `tokenizer_init_kwargs=self.tokenizer.init_kwargs` in _merge_kwargs` as
+        # some init_kwargs are not defined in the forward method of the tokenizer e.g "padding_side"
+        output_kwargs = self._merge_kwargs(
+            OmDetTurboProcessorKwargs,
+            tokenizer_init_kwargs=OmDetTurboProcessorKwargs._defaults["text_kwargs"],
+            # tokenizer_init_kwargs=self.tokenizer.init_kwargs,
```

Comment:
I left this here because it should merge the tokenizer `init_kwargs` to be consistent with other processors, but as we discussed, when I load the model from a saved checkpoint, the tokenizer `init_kwargs` contains kwargs that shouldn't be there such as `padding_side `etc. I'm really not sure what causes this as the checkpoint is saved and loaded similarly to other models.